src/string-indices.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
+
+export default function StringIndices(input, index) {
+  let toArray = input.toString().split(' ')
```

Comment:
I would name this `splitStringToArray` for clarity. Seeing a method that includes the `toString()` method that is called `toArray()` is a bit confusing if a person does not remember quickly that the `string.split()` method returns an array.